app/models/dialog/awx_job_template_dialog_service.rb

```diff
@@ -0,0 +1,175 @@
+class Dialog
+  class AwxJobTemplateDialogService
```

Comment:
Is this a copy of the `AnsibleTowerJobTemplateDialogService` ?  Could we :point_down: 
```suggestion
class Dialog
  class AwxJobTemplateDialogService < Dialog::AnsibleTowerJobTemplateDialogService
  end
end
```

Reply:
Also, tangent but wonder if we could move these to the provider plugins and have everything still work